src/gui.cpp

```diff
@@ -18,7 +18,9 @@
 
 #include "gui.h"
 
+#include <X11/Xmd.h>
```

Comment:
It works great on Linux now, but I doubt Windows would have X11 headers.

Reply:
Ah yes, I've gone through and removed what I believe are unnecessary includes. Looks like something in my IDE is a little over eager when automatically adding them - I'll make sure to look into that before submitting anything more.